### ISSUE TITLE
Queries with Umlauts

### ISSUE THREAD

**Issue by jorol** [LibreCat/Catmandu-SRU#43]
There is a problem with Catmandu::Importer::SRU and queries which contains Umlauts:

```bash
$ curl 'http://services.dnb.de/sru/zdb?version=1.1&operation=searchRetrieve&query=Plön&recordSchema=MARC21-xml'
```

Result: 229 records

```bash
$ catmandu convert SRU --base  http://services.dnb.de/sru/zdb --recordSchema oai_dc --parser simple --query 'Plön' to YAML
```

Result: 0 records

**Comment by nics** [comment]
released as 0.428

**Comment by jorol** [comment]
Thanks @nics!
